apps/network-alerts/test/unit/attention/attention_test.js

```diff
@@ -107,7 +107,6 @@ suite('Network Alerts - Attention Screen', function() {
 
       assert.equal(MockNotifications[0].title, localizedTitle);
       assert.equal(MockNotifications[0].body, body);
-      assert.ok(MockNotifications[0].icon.endsWith('titleID=' + title));
```

Comment:
__nit:__ please add assert that will verify that we passed ```title``` as data property as well